src/css/base.scss

```diff
@@ -2,6 +2,10 @@
 $primary-background: white;
 @import '_reset.scss';
 @import 'flatpickr/dist/flatpickr.css';
+@import '_variables.scss';
+@import '_text.scss';
+@import '_buttons.scss';
+@import '_cards.scss';
 
 body {
```

Comment:
your base.scss or index.scss or *whatever* webpack is being directed to in index.html or scripts.js should *just* have your imports I believe.
Search "Generally, your structure will have the following" [here](https://frontend.turing.edu/lessons/module-2/introduction-to-sass.html)

Reply:
I think the /*...*/ part of the sample structure is where the rest of your base scss code goes. Here's the part of the lesson plan I believe you are referencing:

@import 'Header';
@import 'Button';
@import 'Author';
@import 'Story';
@import 'Stories';
/*...*/
/*...*/